CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting a PR as this can help speed up the process
```

Comment:
`Create`->`Consider creating`, I don't think we want an issue per PR as a hard requirement (e.g. for marking modules as flaky).